src/components/dashboard/DashboardPage.jsx

```diff
@@ -38,36 +38,32 @@ export default function DashboardPage() {
   const userFirstName = useMemo(() => authenticatedUser?.name.split(' ').shift(), [authenticatedUser]);
   const CoursesTabComponent = (
     <>
-      <Container size="lg">
```

Comment:
Just removing this unnecessary `Container` which adds extra padding on the left/right.